sequences/preflight.go

```diff
@@ -138,6 +138,9 @@ func preflight(inpath string) (sequencesPreflight, error) {
 			if z, ok := line.Params["z"]; ok {
 				results.firstLayerZ = float64(z)
 				moveToFirstLayerPointSeen = true
+				// consider first seen "line.IsMoveToFirstLayerPoint()" as a layer change because
+				// we will insert a layer command before it
+				addLookahead(lookaheadLayerChange)
```

Comment:
Do we need to remove the `addLookahead(lookaheadLayerChange)` call below to avoid double-adding lookaheads?